### PR TITLE
Describe offline mode

### PR DESCRIPTION
**Pull request by jason-fox**
Update the documentation to describe how to create an additional custom plugin which offers **offline mode** - this procedure is not recommended for a production environment since it avoids caching of files from the Internet and would be slower than normal online.

Custom plugin because adding it directly would also bloat the plugin codebase with unnecessary duplicates of the Bootstrap files.

### CODE REVIEW

File: sample/offline.dita [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+      <li>
+        <p>In the <filepath>com.example.bootstrap-offline</filepath> root directory, add an Ant script
+            (<filepath>process_offline.xml</filepath>) to predefine various parameters prior to calling the <option
+            >html5-bootstrap</option> transform</p>
+
+          <fig>
+            <title>Sample build file: <filepath>process_offline.xml</filepath></title>
+            <codeblock outputclass="language-xml">&lt;?xml version="1.0" encoding="UTF-8" ?&gt;
+&lt;project
+  name="dita.plugin.net.infotexture.dita-bootstrap.sass"
```
Comment (infotexture) [inline sample/offline.dita:110]: Shall we align this to this agree with the name of the sample plug-in directory above?

```suggestion
  name="dita.plugin.com.example.bootstrap-offline"
```

File: sample/offline.dita [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+  xmlns:if="ant:if"
+  xmlns:unless="ant:unless"&gt;
+  &lt;target name="bootstrap.sass" if="offline.mode"&gt;
```
Comment (infotexture) [inline sample/offline.dita:113]: Shall we use a target name that reflects the purpose?

```suggestion
  &lt;target name="bootstrap.offline" if="offline.mode"&gt;
```

**Review by infotexture — COMMENTED** [review]
👍 Good idea.

At first glance, I thought _“why not build the plug-in instead of just offering instructions”_ — but then I realized we'd need to update it each time Bootstrap releases new versions, so this approach is probably better to reduce maintenance overhead.

**Review by infotexture — COMMENTED** [review]
Found & fixed a few syntax errors in the sample code.

Built the plug-in per instructions, generated output for the sample document, and tested with Wi-Fi disabled.
```console
dita --input=sample/document.ditamap \
     --format=html5-bootstrap \
     --offline.mode=yes
```

Both CSS & JS seem to be included, but ToC navigation doesn't work.

Need to test further with different `nav-toc` values.

**Comment by jason-fox** [comment]
@infotexture - can you describe what you are seeing, as 

```console
dita --input=sample/document.ditamap \
     --format=html5-bootstrap \
     --offline.mode=yes
```

works for me, as do various `--nav-toc` options.

**Comment by infotexture** [comment]
With the default settings (using only the command above), I'm seeing unlinked list items in the ToC. 

Only the current page is a link:

DITA-Bootstrap-offline-ToC-unlinked@2x

If I set `--nav-toc=collapsible`, it seems to work.

I pushed my version of the sample plug-in to https://github.com/infotexture/com.example.bootstrap.offline for reference.

**Comment by jason-fox** [comment]
That matches the defaults currently set within `<target name="dita2html5-bootstrap.init">`:

```xml
<property name="args.html5.toc" value="toc"/>
<property name="nav-toc" value="partial"/>
```

It just so happens that for the dita-bootstrap sample document, the resultant `index.html` document is unnavigable. The same would be true for the base `html5` document

```console
dita --input=sample/document.ditamap \
     --format=html5 \
    --nav-toc=partial
```

You can still use `toc.html` to navigate between files in this case though.

**Review by infotexture — APPROVED** [review]
> It just so happens that for the dita-bootstrap sample document, the resultant `index.html` document is unnavigable. The same would be true for the base `html5` document

I see. Hadn't tried generating output with these defaults yet, so hadn't noticed this up until now, but I see that it's a side effect of the bookman chapter structure in the sample document and not at all related to the offline publishing mode.

Those settings work fine with other documents like the DITA-OT docs, so this is good to go.

**Pull request merged**